README.md

```diff
@@ -129,7 +129,21 @@ If the server list is empty in the headset app:
 
 ## How do I see server logs when using the dashboard?
 
-Click Troubleshoot > Open server logs, or navigate to `${XDG_STATE_HOME}/wivrn/wivrn-dashboard` (with fallback to `${HOME}/.local/state` for `${XDG_STATE_HOME}`, or for flatpak `${HOME}/.var/app/io.github.wivrn.wivrn/.local/state/wivrn/wivrn-dashboard`.
+- Click Troubleshoot > Open server logs, or
+- Navigate to `${XDG_STATE_HOME}/wivrn/wivrn-dashboard` (with fallback to `${HOME}/.local/state` for `${XDG_STATE_HOME}`, or
+- For flatpak, navigate to `${HOME}/.var/app/io.github.wivrn.wivrn/.local/state/wivrn/wivrn-dashboard`.
+
+## I hear crackling audio or see corrupted video signal with an Nvidia GPU
+
+- Switch to NVENC or x264 encoding in Settings, as Vulkan and VAAPI might (will) not work on Nvidia GPUs
+- Try using x264 H.264 instead of NVENC H.264.
```

Comment:
I don't think this is generic enough, per user reports this happens for varied reasons:
* bitrate set way too high (this is the reason for the 200Mb/s cap)
* bad network
* headset plugged in by USB, but using normal network because not started with the specific USB button
* overloaded decoder, due to too high encoded stream

We may replace the paragraph with a note about NVIDIA support, stating that due to lack of developer with such hardware it may not be properly optimized and software encoding or vulkan video encode may offer better performance.

Reply:
The crackling audio section is now weird, with a single item, about software encoding.
I would drop that section as we don't have some relevant advice. We should merge it with the following section which is for generic performance/network issues. The "Try using different encoders" should mention specifically software encoding as a method to check if hardware encoding is not working well.